test/unit/transform/eia923_test.py

```diff
@@ -0,0 +1,53 @@
+"""Unit tests for the pudl.transform.eia923 module."""
+
+import pandas as pd
+
+import pudl.transform.eia923 as eia923
+
+
+def test__yearly_to_monthly_records__normal_values():
```

Comment:
👍🏼 unit tests! I like how you said "Add missing unit test" as if the fact that it was missing stood out. Would it make sense to have a single parameterized test here instead, since the mechanics of the two tests are identical, but the inputs / expected outputs are different? I think you can see the `pudl.analysis.spatial` module for an analogous example.

Reply:
For empty dataframes, I have to use  `pd.testing.assert_frame_equal(..., check_dtype=False)`, because intermediate steps change the dtypes of empty columns. They have no data to go off of, so use default types.